server/src/main/java/com/cloud/network/vpc/VpcManagerImpl.java

```diff
@@ -1983,6 +1995,19 @@ public void doInTransactionWithoutResult(final TransactionStatus status) {
         });
     }
 
+    protected boolean existsVpcDomainRouterWithSufficientNicCapacity(long vpcId) {
+        int countRouterDefaultNics = 2;
+        long countVpcNetworks = _ntwkDao.countVpcNetworks(vpcId);
+        List<DomainRouterVO> vpcDomainRoutersVO = routerDao.listNoRedundantRouterByVpcId(vpcId);
```

Comment:
Why not consider RVR?

Reply:
There was a misunderstanding on my part. I changed the implementation.